components/brave_wallet/browser/simulation_service.cc

```diff
@@ -2,14 +2,14 @@
  * This Source Code Form is subject to the terms of the Mozilla Public
  * License, v. 2.0. If a copy of the MPL was not distributed with this file,
  * You can obtain one at https://mozilla.org/MPL/2.0/. */
-#include "brave/components/brave_wallet/browser/simulation_service.h"
 
 #include <utility>
 
 #include "base/strings/stringprintf.h"
 #include "brave/components/brave_wallet/browser/brave_wallet_constants.h"
 #include "brave/components/brave_wallet/browser/simulation_request_helper.h"
 #include "brave/components/brave_wallet/browser/simulation_response_parser.h"
+#include "brave/components/brave_wallet/browser/simulation_service.h"
 #include "brave/components/constants/brave_services_key.h"
 #include "brave/components/json/rs/src/lib.rs.h"
```

Comment:
Related(to simulation_service.cc) header should come first

https://google.github.io/styleguide/cppguide.html#Names_and_Order_of_Includes

Reply:
fixed